src/composites/FeaturesContainer/Overview/OverviewTab.tsx

```diff
@@ -0,0 +1,143 @@
+import { useEffect, useState } from 'react';
+import { useAtomValue } from "jotai";
+import { State } from "../../../state";
+import { Flex, Text, Box, Button } from "@radix-ui/themes";
+import { HamburgerMenuIcon, Cross1Icon } from '@radix-ui/react-icons';
+import { classifyRiskLevel } from "../../LegendContainer/ClassifyRiskLevel";
+import { BarChart, Bar, XAxis, YAxis, Tooltip, Legend } from 'recharts';
+import { OverviewList } from '.';
+import "./Overview.css"
+
+interface FilterableItem {
+    value: number;
+    weights: Record<string, number>;
+    [key: string]: any;
+}
+
+interface CategoryRiskLevelData {
+    [riskLevel: string]: string[]; // Array of item keys
+}
```

Comment:
equivalent to `Record<string, string[]>`

Reply:
This interface no longer used, but similar values now use Record.